docs/en/for-developers/installing-bhima.md

```diff
@@ -13,60 +13,67 @@ Before you begin the installation process, please make sure you have all the bhi
 sudo apt-get install mysql-server
 
-#Run the following commands to install Redis:
+# Run the following commands to install Redis:
 sudo apt-get install redis-server
 
-#Run the following commands to install curl:
+# Run the following commands to install curl:
 sudo apt-get install curl
 
-#Install node version manager locally
-curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash
+# To find the latest LTS version of nodejs, visit https://nodejs.
+# Then visit https://github.com/nodesource/distributions to find the
+# setup/installer for your OS and the desired version of nodejs.
+curl -fsSL https://deb.nodesource.com/setup_lts.x | sudo -E bash
+sudo apt-get install -y nodejs
 
-#Set up the environmental variables for node version manager
-export NVM_DIR="$HOME/.nvm"
-[ -s "$NVM_DIR/nvm.sh" ] && \."$NVM_DIR\nvm.sh" # This loads nvm
+# Verify that nodejs and npm are installed as expected
+npm --version
+node --version
 
-#Download NodeJS latest long-term version
-nvm install lts/*
-
-#Installs yarn without re-installing NodeJS
+# Installs yarn without re-installing NodeJS
 curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
 echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
 sudo apt-get update && sudo apt-get install yarn --no-install-recommends
-
-#Run the following command to install git:
-sudo apt-get install git
 ```
 
-### Getting dependencies for Puppeteer
+### Install chromium browser ('chrome' on Debian)
+
+```bash
+sudo apt install chromium-browser
+
+# Install extra needed dependencies
+sudo apt-get install libx11-xcb1 libxcomposite1 libasound2 libatk1.0-0 libatk-bridge2.0-0 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgbm1 libgcc1 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6
+```
 
-By default, your OS might not have all dependencies for running puppeteer, the PDF renderer based based on Chromium.  These vary by OS.  For Ubuntu/Debian, you can run the following to prepare your environment:
+We suggest putting the follwing line into your .bashrc file and restarting your shell session (this could also be done in the .env file):
 
 ```bash
-# Install puppeteer dependencies on Ubuntu
-sudo apt-get install ca-certificates fonts-liberation gconf-service \
-  libappindicator1 libasound2 libatk-bridge2.0-0 libatk1.0-0 libc6 libcairo2  \
-  libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgbm1 libgcc1 libgconf-2-4 \
-  libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libnss3 libpango-1.0-0 \
-  libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 \
-  libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 \
-  libxss1 libxtst6 lsb-release libxshmfence1 wget -y
+export CHROME_BIN=`which google-chrome`
```

Comment:
This should match the package name that is `apt install`-ed.   So, I would update this to:
```bash
export CHROME_BIN=`which chromium-browser`
````

Since that is what you installed above.

Reply:
This will work in ~.bashrc,.  I don't think that would work in the `.env` file, since it doesn't execute bash.  Rather, the **output** should be placed in the .env file.  In my case, it is `/usr/bin/chromium-browser`.

I propose you update these instructions to reference the .env _or_ in the .bashrc.